package.json

```diff
@@ -48,22 +49,24 @@
     "eslint-plugin-standard": "^3.0.1",
     "jest": "^19.0.2",
     "prettier": "^1.3.1",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
-    "react-test-renderer": "^15.4.2",
+    "react": "^15.5.0",
+    "react-dom": "^15.5.0",
+    "react-test-renderer": "^15.5.0",
     "semantic-release": "^6.3.6"
   },
   "peerDependencies": {
-    "react": "^15.0.0",
-    "patternfly": "~3.3"
+    "classnames": "^2.2.5",
+    "immutability-helper": "^2.2.0",
+    "prop-types": "^15.5.10",
+    "react": "^15.5.0"
   },
   "scripts": {
     "commit": "git-cz",
     "build": "babel src --out-dir lib --ignore test.js",
     "lint": "eslint --max-warnings 0 src storybook",
     "prettier": "prettier --write --single-quote --no-semi '{src,storybook}/**/*.js'",
     "prepublish": "npm run build",
-    "test": "npm run lint && jest",
+    "test": "rm -rf ./lib && npm run lint && jest",
```

Comment:
Do you hit any issue without this? I am not able to reproduce any problem without.